airbyte-integrations/connectors/source-salesforce/setup.py

```diff
@@ -5,7 +5,7 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk==0.1.79", "vcrpy==4.1.1", "pandas"]
+MAIN_REQUIREMENTS = ["airbyte-cdk~=0.1.79", "vcrpy==4.1.1", "pandas"]
```

Comment:
It seems airbyte fixed the issue we found.
https://github.com/rudderlabs/airbyte/pull/21